src/icpp/commands_build_wasm.py

```
@@ -365,34 +368,56 @@ def c_compile_file_mine(file: str) -> None:
+            "         or pass `--generate-bindings no` to silence this."
+        )
+        return
+
+    out_path = declarations_path / canister
+    out_path.mkdir(parents=True, exist_ok=True)
+    typer.echo(f"{out_path.resolve()}")
+
+    try:
+        shutil.copy(did_path, out_path / f"{canister}.did")
+        for target, suffix in (("js", "did.js"), ("ts", "did.d.ts")):
+            cmd = f"didc bind {str(did_path)} --target {target}"
```

Ruff (0.15.21)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)